client/src/pages/AirTableCOVID.js

```diff
@@ -1,4 +1,4 @@
-import React, { Component } from "react";
+import React, { useEffect, useState } from "react";
```

Comment:
I believe the `/AirTableCOVID` page will be gone based on discussion on Slack. Were there changes?

Reply:
No idea. Should It remove it then?

Reply:
Yea let's remove it. Currently the data is directly from Airtable, and we want to eventually use just our backend for serving data. We're in the midst of migrating data from Airtable to our database. Also, that data are Posts, which will be shown in the Feed page instead. So there's not much use for this page.

Reply:
removed